src/createAnimatedComponent.tsx

```diff
@@ -606,11 +634,31 @@ export default function createAnimatedComponent(
     }
 
     componentDidUpdate(
-      prevProps: AnimatedComponentProps<InitialComponentProps>
-    ) {
+      prevProps: AnimatedComponentProps<InitialComponentProps>,
+      prevState: Readonly<unknown>,
+      snapshot?: any
+    ): void {
       this._reattachNativeEvents(prevProps);
       this._attachAnimatedStyles();
       this._attachInlineProps();
+
+      if (IS_WEB && snapshot !== null) {
```

Comment:
Comment added in [15a30ed](https://github.com/software-mansion/react-native-reanimated/pull/4805/commits/15a30edc40b46067703120256f8c446b2cd9b46e). And yes, passing null is necessary - otherwise you'll get error in console